templates/templates_config/config_tags.html.tt2

```diff
@@ -47,4 +47,22 @@
         <br> When tagging archives using Plugins, these tags will be automatically omitted.
         <br>Split tags with commas.
     </td>
-</tr>
\ No newline at end of file
+</tr>
+
+<tr>
+    <td class="option-td">
+        <h2 class="ih"> Tag Rules </h2>
+    </td>
+    <td class="config-td">
+        <input id="tagsruleson" name="tagsruleson" class="fa" type="checkbox" [% IF tagsruleson %]checked[% END %]>
+        <br>
+        <textarea class="stdinput" size="20" style='width:100%; height:75px' name="tagsrules"
+            maxlength='5000'>[% tagsrules %]</textarea>
+        <br>
```

Comment:
It might also be good to add a page to the documentation for this feature later:  
https://github.com/Difegue/LANraragi/tree/dev/tools/Documentation/advanced-usage  
(Blacklisting didn't have a page, but then again it was a much simpler feature 😅)